### ISSUE TITLE
FR: Do not lint URLs

### ISSUE THREAD

**Issue by pcorpet** [amperser/proselint#759]
In one of my .md files, I have something like:
```md
Please read the [report from June](http://www.example.com/reports/june).
```
Proselint detects that "june" in the URL is not properly spelled and should be "June" in my URL. However I would prefer proselint to ignore it altogether. I'm not sure how to fix it generically, but I think I'm not the only one running into this problem.

**Comment by rriemann** [comment]
In some cases, the link may be relative, e.g. 

    Please read the [report from June](reports/june).

So I do not have an idea for an easy rule.

**Comment by KVGarg** [comment]
Hey @suchow , what you think regarding this issue? I think adding a rule over `urls` will solve this issue because if not added then most of the `checks` will raise warning, for example - say a `url` include month or annotations or any other keyword which are provided in `checks *.py` files it will definitely raise a warning which  should be avoided.

**Comment by KVGarg** [comment]
I have added a precondition in tools.py to avoid linting over urls, but haven't created a PR.
I will a create a PR, if you guys think that this is of worth.

**Comment by pieterocp** [comment]
@KVGarg that would be appreciated since you cannot easily control the syntax of third parties urls, e.g.

```error
./core/messenger.md:111:61: garner.phrasal_adjectives.ly No hyphen is necessary in phrasal adjectives with an adverb ending in -ly, unless the -ly adverb is part of a longer phrase
```

```md
Handler ordering can be configured [using messenger priority tag](https://symfony.com/doc/current/messenger.html#manually-configuring-handlers).
```
